src/components/PopupConfirm.js

```diff
@@ -0,0 +1,22 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._imageEditForm = this._popupElement.querySelector(".modal__form");
```

Comment:
Needs correction: This is a general class for confirmation, so it should not know anything about the specific entities, such as image edit. Please rename this variable to `form`